javascript/javascript1/week1/dog_age_calculator.js

```diff
@@ -0,0 +1,21 @@
+//A dog age calculator
+ const dogYearOfBirth=2017;
+ const dogYearFuture=2027;
+ let dogYear= dogYearFuture-dogYearOfBirth;
+//const humanYear=dogYearFuture-dogYearOfBirth;
+ShowResultInDogYears=true; //boolean type of variable
```

Comment:
Remember to declare your variables by using either `const` or `let`.

Reply:
[const explained](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const)